src/components/pickup/Pickedup.tsx

```diff
@@ -0,0 +1,15 @@
+import React, {useEffect, useState} from "react";
+import PickupTable from "./PickupTable";
+import { Order } from "../../types/Order";
+
+function Pickedup({ orders }: { orders: Order[] }) {
```

Comment:
This is a pretty small file. Is it really useful? Why not just move filter and `PickupTable` into the parent?

Reply:
I would make an argument for segmenting components where there is a natural distinction. The PickedUp tab is a separate tab, and leaving it in a separate file declutters the parent component. A rule that I like to follow is a component should aim to be smaller than 70 lines. Otherwise, it starts to have too many responsibilities, so its hard to maintain. Its a small component for sure, but it's small win to reduce the parent.

I do think it adds semantic value too, for from the perspective of Tabs.tsx, it's immediately clear that `<PickedUp>` is the tab containing picked up orders. I think that reads better than `<PickedUpTable orders={pickedUpOrders}/>`.

Now that the component has been refactored, the PickUp tab doesn't really serve any value in being separate. However, it is follow the pattern set by the PickedUp tab, so it does serve some semantic value too.

Reply:
On a file size basis, this is only saving 2 lines in the parent and adding a bunch of boilerplate to the codebase as a whole so it's not saving lines. The semantics argument makes sense. 

I'll leave it up to you